docs/contributing.md

```diff
@@ -20,15 +20,15 @@ This means that in order to contribute, you need a GitHub account. Read more abo
 
 ## Types of Contribution
 
-As mentioned earlier, there are two ways in which you can contribute to SAP documentation - either by providing feedback, or by contributing content.
+As mentioned earlier, there are two ways in which you can contribute to SAP documentation - either by providing feedback, or by contributing content. Both ways are available via the *Feedback* link near the top of the page.
 
 ### Providing Feedback
 
 We value feedback on our documentation, and want to make it possible for you to do that in a simple but structured way.
 
-Where feedback is possible for a page, you'll see a link inviting you to provide feedback.
+Where feedback is possible for a page, you'll see a *Create issue* option in the menu shown when you select the *Feedback* link.
```

Comment:
```suggestion
Where feedback is possible for a page, you'll see a *Create issue* option in the *Feedback* menu.
```

Reply:
I'd consciously avoided "menu" because it's not really a menu, it's (at most) a "menu option", but that would then leave me fewer choices to use to refer to the children. But I don't mind making this change.

Reply:
Do you mean it's technically not a menu? When we discussed the UI changes with Philippe, we kept calling it menu. 

I agree that as soon as you click on it, it looks like a link (didn't see that during my review, sorry), but the behavior is the same as for menus, right? You click on it, and the different options appear. Those options look like links again, not menu entries, I think I see what you mean now.

To me, this UI element is mimicking a menu by means of browser technology. Users do exactly the same as what they'd do for menus: One click to open it, another click to choose the option.

If we want to avoid calling it something altogether we could say:

> Choose *Feedback* > *Create issue* to provide feedback. If the option is not available the guide does not take part in the initiative yet.

One argument to use "Choose" is that it is device independent (compared to "click") - that's not coming from me, but from our documentation guidelines.

Reply:
I love this level of detailed discussion! It's a link, an unstyled link at that (which does break general Web usability guidelines, but we'll ignore that for now), which is why I called it a link originally. Even if you hover over it, the underline appears, giving the user the signal (too late, really) that it's supposed to be a link. 

I always avoid using hardware-device dependent verbs, which is the reason I don't use "click" here, yes.

I like the *Parent > Child* approach, and will make the changes to adopt that, and avoid exacerbating things by trying to frame it with words ;-)